Public/Src/Engine/UnitTests/Scheduler/PipFingerprinterTests.cs

```diff
@@ -66,6 +67,16 @@ internal static PipFragmentRenderer.ContentHashLookup GetContentHashLookup(FileA
             };
         }
 
+        internal static PipFragmentRenderer.ContentHashLookup GetDummyContentHash(int randomNumber)

```

Comment:
Don't copy-paste.  Extent the existing `GetContentHashLookup` function to accept this `randomNumber` argument (there you can set its default value to 1 so that you don't have to update any other callers of `GetContentHashLookup`